README.md

```diff
@@ -1,9 +1,99 @@
-# Separation of Concerns Starter
+# Todo List
 
-<!-- describe your project -->
+> A todo list allows the user to add, edit, check and delete a list of items   .
 
----
+## Table of contents
+
+- [Todo List](#todo-list)
+  - [Table of contents](#table-of-contents)
+  - [General info](#general-info)
+  - [Screenshots](#screenshots)
+  - [Technologies](#technologies)
+  - [Code Examples](#code-examples)
+  - [Features](#features)
+  - [Status](#status)
+  - [Inspiration](#inspiration)
+  - [Contact](#contact)
+
+## General info
+
+A todo list , the objective is build the app using Javascript to create DOM elements during the user interaction with the app interface .
```

Comment:
This group project is part of [HackYourFuture Belgium](https://hackyourfuture.be/) module **Separation of concerns**. Our objective is to build an app using Javascript to create DOM elements during the user interaction with the app interface .